contracts/SecurityTokenRegistry.sol

```diff
@@ -35,10 +35,15 @@ contract SecurityTokenRegistry is ISecurityTokenRegistry, Util, Pausable, Regist
 
     // Emit when changePolyRegistrationFee is called
     event LogChangePolyRegistrationFee(uint256 _oldFee, uint256 _newFee);
-
     // Emit at the time of launching of new security token
-    event LogNewSecurityToken(string _ticker, address indexed _securityTokenAddress, address indexed _owner);
-    event LogAddCustomSecurityToken(string _name, string _symbol, address _securityToken, uint256 _addedAt);
+    event LogNewSecurityToken(
+        string _ticker,
+        string _name,
+        address indexed _securityTokenAddress,
```

Comment:
Is there any reason why address of the token should be indexed? @satyamakgec @VictorVicente

Reply:
I don't remember exactly but that was the requirement of the dApps team @bshevchenko

Reply:
We'll leave it.